core/src/main/java/com/digitalpebble/stormcrawler/filtering/URLFilter.java

```diff
@@ -26,7 +26,7 @@
  *
  * @see URLFilters for more information.
  */
-public interface URLFilter extends Configurable {
+public abstract class URLFilter extends AbstractConfigurable {
```

Comment:
I guess, that it is in the nature of this change, that we are breaking user code, right? So we should mention that in the release notes.

Reply:
yes, we will